src/main/app/claims/models/claim.ts

```diff
@@ -89,7 +89,16 @@ export class Claim {
     if (!this.respondedAt) {
       return undefined
     }
-    return this.respondedAt.clone().add('33', 'days')
+    const daysForService = 5
+    const daysForResponse = 28
```

Comment:
should these perhaps be global variables? they must be used some where else in the application? I found one other instance in paymentDateChecker.ts

Reply:
I guess 5 days for service might be global variable but it is being used only once so I did not bother. 28 days mentioned in `paymentDateChecker.ts` has a different meaning.